src/bootstrap.php

```diff
@@ -408,7 +408,7 @@ private function register_scripts() {
 			'wp-color-picker',
 		];
 
-		wp_register_script( 'gfpdf_js_settings', PDF_PLUGIN_URL . 'dist/assets/js/gfpdf-settings.min.js', $pdf_settings_dependancies, $version );
+		wp_register_script( 'gfpdf_js_settings', PDF_PLUGIN_URL . 'dist/assets/js/gfpdf-settings-main.min.js', $pdf_settings_dependancies, $version );
```

Comment:
Lets rename is `admin.js` and change the folder from `gfpdf-settings` to `admin`